migration/migrating_3_4/planning-migration-3-to-4.adoc

```diff
@@ -72,7 +71,7 @@ For more information, see xref:../../architecture/architecture-installation.adoc
 [discrete]
 ==== Upgrading your cluster
 
-In {product-title} 3.11, you upgraded your cluster by running Ansible playbooks. In {product-title} 4.6, the cluster manages its own updates, including updates to {op-system-first} on cluster nodes. You can easily upgrade your cluster by using the web console or by using the `oc adm upgrade` command from the OpenShift CLI and the Operators will automatically upgrade themselves. If your {product-title} 4.6 cluster has Red Hat Enterprise Linux worker machines, then you will still need to run an Ansible playbook to upgrade those worker machines.
+In {product-title} 3.11, you upgraded your cluster by running Ansible playbooks. In {product-title} 4.6, the cluster manages its own updates, including updates to {op-system-first} on cluster nodes. You can easily upgrade your cluster by using the web console or by using the `oc adm upgrade` command from the OpenShift CLI and the Operators will automatically upgrade themselves. If your {product-title} 4.6 cluster has RHEL worker machines, then you will still need to run an Ansible playbook to upgrade those worker machines.
```

Comment:
I don't know if MTC uses `modules/common-attributes.adoc`, but it has variables for RHEL:

```
:op-system-base: RHEL
:op-system-base-full: Red Hat Enterprise Linux (RHEL)
```